src/read_model/SGJavaScriptBridge.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @callback dispatchCommandsStringForVersion
+ * @param {string} command
+ * @param {string} version
+ */
+
+/**
+ * @callback dispatchCommandsForVersion
+ * @param {Object[]} command
+ * @param {string} version
+ */
+
+/**
+ * @typedef {Object} window.SGJavaScriptBridge
+ * @property {dispatchCommandsStringForVersion} dispatchCommandsStringForVersion
+ * @property {dispatchCommandsForVersion} dispatchCommandsForVersion
+ */
+window.SGJavascriptBridge = Object();
```

Comment:
I couldn't get it working for me because I use only type based auto completion. The file is not deployed within our code therefore this shouldn't cause any issues to just leave it as it is.